src/main/java/ddingdong/ddingdongBE/domain/filemetadata/service/FileMetaDataServiceImpl.java

```diff
@@ -21,42 +23,45 @@ public class FileMetaDataServiceImpl implements FileMetaDataService {
 
     @Override
     @Transactional
-    public void save(List<FileMetaData> fileMetaDataList) {
-        List<FileMetaData> newFileMetaDataList = fileMetaDataList.stream()
-                .filter(fmd -> !fileMetaDataRepository.existsById(fmd.getId()))
-                .toList();
-
-        if (!newFileMetaDataList.isEmpty()) {
-            fileMetaDataRepository.saveAll(newFileMetaDataList);
-        }
-    }
-
-    @Override
-    @Transactional
-    public UUID save(FileMetaData fileMetaData) {
+    public UUID create(FileMetaData fileMetaData) {
         FileMetaData savedFileMetaData = fileMetaDataRepository.save(fileMetaData);
         return savedFileMetaData.getId();
     }
 
     @Override
-    public FileMetaData getById(UUID id) {
-        return fileMetaDataRepository.findById(id)
-                .orElseThrow(() -> new ResourceNotFound("FileMetaData(id=" + id + ")를 찾을 수 없습니다."));
-    }
-
-    @Override
-    public List<FileMetaData> findActivatedAll(DomainType domainType, Long entityId) {
+    public List<FileMetaData> getCoupledAllByEntityTypeAndEntityId(DomainType domainType, Long entityId) {
```

Comment:
```suggestion
    public List<FileMetaData> getCoupledAllByDomainTypeAndEntityId(DomainType domainType, Long entityId) {
```
DomainType으로 통일시키는 것이 좋아보입니다!